tests/src/test/java/io/streamnative/pulsar/handlers/kop/SchemaRegistryTest.java

```diff
@@ -47,18 +46,10 @@
 @Slf4j
 public class SchemaRegistryTest extends KopProtocolHandlerTestBase {
 
-    private String bootstrapServers;
+    protected String bootstrapServers;
```

Comment:
Why do you change it from `private` to `protected`?

Reply:
This is because I have many "Proxy" tests that inherit from the original tests and override the address of the Kafka broker.
See here
https://github.com/datastax/starlight-for-kafka/blob/2.8.0_ds/tests/src/test/java/io/streamnative/pulsar/handlers/kop/SchemaRegistryProxyTest.java#L32

if this is a problem for you I can keep "private"

Reply:
It's not a big deal. Since the proxy is not accepted by SN at the moment, we can use the `protected` keyword here unless someone want to change it in future.